packages/forms/stories/14-FileUpload.stories.tsx

```diff
@@ -29,54 +27,6 @@ export default {
   component: FileUpload
 } as Meta;
 
-const StyledFile = styled(File)`
-  .progress-bar {
-    &.enter {
-      opacity: 0;
-    }
-
-    &.enter-active {
-      transition: opacity 200ms;
-      opacity: 1;
-    }
-
-    &.exit {
-      opacity: 1;
-    }
-
-    &.exit-active {
-      transition: opacity 200ms;
-      opacity: 0;
-    }
-
-    &.exit-done {
-      display: none;
-    }
-  }
-`;
-
-const StyledSpan = styled(Span)`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  width: 100%;
-`;
-
-const StyledProgress = styled(Progress)`
-  position: absolute;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  margin: 0;
-`;
-
-const StyledEllipsis = styled(Ellipsis)`
-  /* stylelint-disable-next-line property-no-unknown */
-  margin-${props => (props.theme.rtl ? 'left' : 'right')}: ${props => props.theme.space.base * 2}px;
-  min-width: 200px;
-  max-width: 300px;
-`;
```

Comment:
This styling needs to ship with the component in order to keep consumer implementations DRY.